include/http/request_handler.js

```diff
@@ -356,6 +356,20 @@ module.exports = function RequestHandlerModule(pb) {
                 }
 
                 this.resp.setHeader('content-type', contentType);
+
+                const secureCookies = this.req && this.req.siteObj && this.req.siteObj.secureCookies;
+                if (secureCookies) {
+                    const cookies = this.resp.getHeader('set-cookie');
+                    (cookies || []).forEach((cookie, index) => {
+                        if (!/\bsecure\b/g.test(cookie)) {
+                            cookies[index] = `${cookie}; Secure`;
+                        }
+                    });
+                    if (cookies) {
```

Comment:
here also, if we do the safety check beforehand, you don't need the 'if(cookies)' check here because you have checked before(e.g. cookies.length === 0, return ) and if the code goes here, the cookies is 100% non-empty.

Reply:
Since we need to check the cookie twice. I agree with Jack, there is no need to check the cookie twice here.

Reply:
thanks Jack and Shenghua for your support!!
I made the changes. Kindly review again.
Thanks in advance!!